### PR TITLE
Mute phone on lesson beginn

### LINKED ISSUE #29: [Feature-Request] Automatically mute the smartphone when a lesson starts
This would be a nice to have feature.

IIRC the official app has this feature too.

### PR DESCRIPTION
**Pull request by SimonIT**
This PR closes #29 the feature request of muting the phone at lesson beginn. I added also a notification before the first lesson. The time can be changed by the user. 

I removed some duplicate methods.

I suggest to squash my commits because I made too much of them

### CODE REVIEW

File: app/src/main/java/de/perflyst/untis/activity/ActivityMain.java [hunk 1]
```diff
@@ -26,51 +26,28 @@
...
-import android.view.View;
-import android.view.ViewGroup;
-import android.widget.GridView;
-import android.widget.ImageView;
-import android.widget.LinearLayout;
-import android.widget.RelativeLayout;
-import android.widget.TextView;
-
+import android.view.*;
+import android.widget.*;
```
Comment (Perflyst) [inline app/src/main/java/de/perflyst/untis/activity/ActivityMain.java]: I know this is was automatically done. I am still not sure if this is better or worse.

File: app/src/main/java/de/perflyst/untis/activity/ActivityMain.java [hunk 2]
```diff
@@ -26,51 +26,28 @@
...
 import java.text.SimpleDateFormat;
-import java.util.ArrayList;
-import java.util.Calendar;
-import java.util.Collections;
-import java.util.List;
-import java.util.Locale;
-import java.util.NoSuchElementException;
-
-import de.perflyst.untis.utils.ThemeUtils;
+import java.util.*;
```
Comment (Perflyst) [inline app/src/main/java/de/perflyst/untis/activity/ActivityMain.java]: Same here as above, should we keep it?

Comment (SimonIT) [reply app/src/main/java/de/perflyst/untis/activity/ActivityMain.java]: What's your point about not using *? In my opinion the import block shrinks which reduces the file size.
[Here](https://coderanch.com/wiki/659753/Wild-Card-Specific-Imports) are some arguments for and against wildcard imports

Comment (Perflyst) [reply app/src/main/java/de/perflyst/untis/activity/ActivityMain.java]: Well, ok. As long as there is no other library who may conflict with these we can keep it.

**Comment by Perflyst** [comment]
Thank you, I will create a new release now

**Pull request merged**